src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
```

Comment:
The 'getRandomBall()' function is called without checking if it returns a valid object. It's a good practice to ensure that the function does not return 'null' before using its returned object.

Reply:
done